frontend/express/public/javascripts/countly/vue/components/input.js

```diff
@@ -552,8 +561,10 @@
             innerValue: {
                 get: function() {
                     if (this.uncommittedValue && this.uncommittedValue !== this.value) {
+                        this.selectedCountText = CV.i18n('export.export-columns-selected-count', this.uncommittedValue.length, this.options.length);
```

Comment:
You can define a computed selectedCountText property for this, instead of adding a side effect to getter of innerValue